odbc_fdw.c

```diff
@@ -415,6 +417,27 @@ odbc_connection(odbcFdwOptions* options, SQLHENV *env, SQLHDBC *dbc)
 	ret = SQLDriverConnect(*dbc, NULL, (SQLCHAR *) conn_str.data, SQL_NTS,
 	                       OutConnStr, 1024, &OutConnStrLen, SQL_DRIVER_COMPLETE);
 	check_return(ret, "Connecting to driver", dbc, SQL_HANDLE_DBC);
+	elog_debug("connection opened >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
+}
+
+/*
+ * Close the ODBC connection
+ */
+static void
+odbc_disconnection(SQLHENV *env, SQLHDBC *dbc)
+{
+	if (dbc && *dbc)
```

Comment:
This is suspicious and likely what's causing crashes on Windows builds. You are setting the contents of `dbc` to NULL, but is `SQLHDBC` declared as a NULLable type in the ODBC standard (I'm guessing it comes from there)? 

In any case, the fact that we are asking ourselves this question means that this approach is wrong. Either always use a pointer or ensure you only call the disconnect once so you don't need the NULL guards.